server/db/seeds/development/seed-admins-table.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  async seed(knex) {
+    // Deletes all existing entries
+    await knex('admins').del()
+
+    // Insert seed entries
+    await knex('admins').insert([
+      {
+        id: 1,
```

Comment:
For `id`, `active`, `created_at`, `updated_at`, let's omit those values from the seed since they are auto-generated. On my local machine, when I pulled this code to test it out, setting the id on the seed de-synchronizes the id counter on the table so when I try to insert a new admin, it tries to use the value `id: 1` and has a problem.